rust/headless-client/src/ipc_service.rs

```diff
@@ -359,13 +359,32 @@ impl<'a> Handler<'a> {
     async fn run(&mut self, signals: &mut signals::Terminate) -> HandlerOk {
         loop {
             match poll_fn(|cx| self.next_event(cx, signals)).await {
-                Event::Callback(x) => {
-                    if let Err(error) = self.handle_connlib_cb(x).await {
-                        tracing::error!(
-                            error = anyhow_dyn_err(&error),
-                            "Error while handling connlib callback"
-                        );
-                        continue;
+                Event::Callback(msg) => {
+                    let mut backoff = backoff::ExponentialBackoffBuilder::new()
+                        .with_initial_interval(Duration::from_millis(100))
+                        .with_max_elapsed_time(Some(Duration::from_secs(10)))
+                        .build();
+
+                    loop {
+                        let error = match self.handle_connlib_msg(msg.clone()).await {
+                            Ok(()) => {
+                                tracing::info!(?msg, "Successfully handled msg from connlib");
```

Comment:
Probably want this at `debug` log level, no? We'll see tons of these during normal operation if the network change listener is still noisy.